official/recommendation/data_preprocessing.py

```diff
@@ -392,9 +392,10 @@ def _shutdown(proc):
   tf.logging.info("Shutting down train data creation subprocess.")
   try:
     try:
-      proc.send_signal(signal.SIGINT)
```

Comment:
I'm reluctant to get rid of this. If a sigint is sent, the subprocess can catch the KeyboardInterrupt and exit gracefully. terminate() doesn't afford that luxury.

Reply:
Done. I thought changing this was required, but it's not.